app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -203,23 +211,46 @@ class Identification extends React.Component {
             message +=
               ' is supported for email and SMS text reporting methods only. If telephone call is selected as the preferred reporting method, the call will be in English.';
           }
-          return (
-            <i>
-              <b>* Warning:</b> {message}
-            </i>
-          );
         }
       }
     }
-  }
+    return message;
+  };
+
+  /**
+   *
+   * @param {*} isPrimary - whether to render the primary or secondary language Select
+   */
+  renderLanguageSelect = isPrimary => {
```

Comment:
Love breaking this out!!

Reply:
This is a lot cleaner than how it was before with the different language vars

Reply:
+1